src/WebKit_Performance_001.test.js

```diff
@@ -0,0 +1,88 @@
+import {
+  pluginDeactivate,
+  pluginActivate,
+  webKitBrowserActions,
+  settingUrl,
+} from './commonMethods/commonFunctions'
+import {
+  validateFPS,
+  initializeSamplesArray,
+  fetchFPS,
+  calcAvgFPS,
+} from './commonMethods/webKitPerformanceCommonFunctions'
+import constants from './commonMethods/constants'
+
+const URL = 'http://webkit.org/blog-files/3d-transforms/poster-circle.html'
+let minFPS = 40
+
+export default {
+  setup() {
+    initializeSamplesArray()
+  },
+  title: 'WPEWebkit performance poster circle',
+  description: 'Loads the Poster Circle CSS3 animation and measures its performance',
+  steps: [
+    {
+      description: 'Deactivating monitor',
+      test: pluginDeactivate,
+      params: constants.monitorPlugin,
+      assert: 'deactivated',
+    },
+    {
+      description: 'Deactivating WebKitBrowser',
+      test: pluginDeactivate,
+      params: constants.webKitBrowserPlugin,
+      assert: 'deactivated',
+    },
+    {
+      description: 'Activating WebKitBrowser',
+      test: pluginActivate,
+      params: constants.webKitBrowserPlugin,
+      assert: 'suspended',
+    },
+    {
+      description: 'Resuming WebKitBrowser',
+      test: webKitBrowserActions,
+      params: constants.resume,
+      assert: 'resumed',
+    },
+    {
+      description: 'Navigating to Poster Circle URL',
+      test: settingUrl,
+      params: URL,
+      assert: URL,
+    },
+    {
+      description: 'Sleeping for 5 seconds',
```

Comment:
instead of sleeping for a fixed 5s can we listen for the on Loaded event on the WebKit browser to check if the URL is properly loaded? 

The reason is an RPI3 might load that url just fine in 5 seconds, but really low end MIPS boxes might not. Though granted, they'll likely not meet the required FPS.

Reply:
See this test for an example of using a listener: https://github.com/WebPlatformForEmbedded/Storm-Testcases/blob/master/src/appstore-weather-network.test.js

Also, in general there is no need to make a separate "sleep-step". If you add a `sleep`-key to a step, it will wait the specified amount _before_ executing the `test()`-method.

In this case I imagine it's because you don't want to sleep every repetition of the 'Fetch FPS' step. Perhaps we should add some kind of `sleepOnce`-functionality to Storm for steps that repeat, @wouterlucas?

Reply:
Yea I think the seperate sleep steps where added for UI/UX workarounds, as it may be confusing to a tester/user if the test sits on fetchFPS for a while before it actually executes it. I might be to blame for that and took the quick approach.

I agree that sleep should be part of the test step itself. Albeit we should also reflect in the CLI/Frontend UIs that the test isnt being executed but we're waiting for a sleep to finish. 

And yes @michielvandergeest I agree sleepOnce makes sense.

Reply:
I have added a listener in setup to address this comment